src/explore-education-statistics-admin/src/services/releaseService.ts

```diff
@@ -27,6 +31,7 @@ export interface Release {
   timePeriodCoverage: ValueLabelPair;
   title: string;
   type: ReleaseType;
+  contact?: Contact;
```

Comment:
We shouldn't need this. We removed this from `ReleaseViewModel` in a PR #4299

Reply:
Cool, it is purged now thanks